addons/web_editor/controllers/main.py

```diff
@@ -56,7 +56,7 @@ def export_icon_to_png(self, icon, color='#000', bg=None, size=100, alpha=255, f
         # Make sure we have at least size=1
         size = max(1, min(size, 512))
         # Initialize font
-        addons_path = http.addons_manifest['web']['addons_path']
+        addons_path = load_manifest('web')['addons_path']
         font_obj = ImageFont.truetype(addons_path + font, size)
```

Comment:
The weird-ass path manipulation seems unnecessary as `truetype` supports file objects, just use `file_open(font)`.